app/views/govuk_publishing_components/components/docs/attachment.yml

```diff
@@ -54,3 +54,12 @@ examples:
         content_type: text/plain
         file_size: 108515
       hide_help_text: true
+  with_contact_email:
+    data:
+      attachment:
+        title: "Department for Transport information asset register"
+        url: https://assets.publishing.service.gov.uk/government/uploads/system/uploads/attachment_data/file/747661/department-for-transport-information-asset-register.csv
+        filename: department-for-transport-information-asset-register.csv
+        content_type: application/pdf
+        file_size: 20000
+      contact_email: defra.helpline@defra.gsi.gov.uk
```

Comment:
This should be part of the attachment block of data, as that is data that would be included in rendering this via govspeak

Reply:
I was wondering about that. Is the attachment block mirroring some 'model' structure? or is it freestyle in terms of naming things?

Reply:
Ideally it should follow the precedent set by existing attachment data sent to govspeak to avoid unnecessary breaks in consistency and backwards compatibility. However the previous way is mostly Whitehall internal data and there is not a backing of a schema or similar to enforce it.

Ideally in the future they should be done by a schema though.

The Whitehall naming is alternative_format_contact_email: https://github.com/alphagov/whitehall/blob/5c760eea912320c5a80ef59791df47161d889f2f/app/helpers/govspeak_helper.rb#L22-L24

It doesn't look like this was ever fully implemented into govspeak though, as a single address was hardcoded: https://github.com/alphagov/govspeak/blob/f626b20e9e0a742cc4c9f4a946a80c87aa63a8da/lib/govspeak/presenters/attachment_presenter.rb#L102-L104

Reply:
Thanks for pointing to those. The Whitehall bit was missing from my puzzle.